Content.Server/Chemistry/ReagentEffects/Paralyze.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using Content.Shared.Chemistry.Reagent;
+using Content.Server.Stunnable;
+
+namespace Content.Server.Chemistry.ReagentEffects;
+
+public sealed class Paralyze : ReagentEffect
+{
+    [DataField("paralyzeTime")] public double ParalyzeTime = 2;
+
+    /// <remarks>
+    ///     true - refresh paralyze time,  false - accumulate paralyze time
+    /// </remarks>
+    [DataField("refresh")] public bool Refresh = true;
+
+    public override void Effect(ReagentEffectArgs args)
+    {
+        EntitySystem.Get<StunSystem>().TryParalyze(args.SolutionEntity, TimeSpan.FromSeconds(ParalyzeTime), Refresh);
+
+        args.Source?.RemoveReagent(args.Reagent.ID, args.Quantity); // Remove reagent, so that, presumably, it doesn't reapply 317589179857 times(?)
```

Comment:
do not do this, metabolism already handles this

Reply:
unless the behavior you're looking for is "stuns once and then deletes itself and doesnt do it again" in which case, i dont really see why it should do that?

Reply:
should be good now